### ISSUE TITLE
Controlling Access to the Kubernetes API

### ISSUE THREAD

**Issue by hidalgopl** [kubernetes/website#29300]
Since https://github.com/kubernetes/kubernetes/pull/95856 is merged, following paragraph:
```markdown
By default, the Kubernetes API server serves HTTP on 2 ports:

localhost port:

is intended for testing and bootstrap, and for other components of the master node (scheduler, controller-manager) to talk to the API
no TLS
default is port 8080, change with --insecure-port flag.
default IP is localhost, change with --insecure-bind-address flag.
request bypasses authentication and authorization modules.
request handled by admission control module(s).
protected by need to have host access
```
[here](https://kubernetes.io/docs/concepts/security/controlling-access/#api-server-ports-and-ips)
is no longer true

**Comment by simanw** [comment]
Hey hidalgopl, I can work on this. I reviewed #95856. If I get it right, removing "change with --insecure-port flag" is enough. Please let me know if I miss anything.
/assign

**Comment by hidalgopl** [comment]
> Hey hidalgopl, I can work on this. I reviewed #95856. If I get it right, removing "change with --insecure-port flag" is enough. Please let me know if I miss anything.
> /assign

sounds good, thank you :+1:

**Comment by jihoon-seo** [comment]
This issue seems okay to be closed, since the PR #29447 is merged.
/close
